src/seedu/addressbook/Main.java

```diff
@@ -109,7 +109,12 @@ private CommandResult executeCommand(Command command)  {
         try {
             command.setData(addressBook, lastShownList);
             CommandResult result = command.execute();
-            storage.save(addressBook);
+            try {
```

Comment:
Is there any point having a `try-catch` block inside a `try-catch` block?

Tip: This is possible

```
try { 
...
} catch(StorageOperationException e) {
...
} catch (Exception e) {
... 
}
```

Google to learn which excpetion is caught in which block. (It was last semester's final qn in MCQ)